src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -243,6 +244,19 @@ export class MoreInfoDialog extends LitElement {
     this.closeDialog();
   }
 
+  private _handleTabChanged(ev: CustomEvent): void {
+    const newTab = ev.detail.index;
+    if (newTab === this._currTabIndex) {
+      return;
+    }
+
+    this._currTabIndex = ev.detail.index;
+    if (!this._historyImported) {
+      import("./ha-more-info-tab-history");
```

Comment:
Let's import this in `firstUpdated`. That will make sure the dialog is loaded and rendered fast, and that if you go to the history tab it is also there immediately.